rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_13.json

```diff
@@ -165,7 +166,19 @@
                                                         "type": "VARIABLE_AIR_VOLUME",
                                                         "served_by_heating_ventilating_air_conditioning_system": "System 7",
                                                         "heating_source": "HOT_WATER",
-                                                        "heating_from_loop": "Boiler Loop 1"
+                                                        "heating_from_loop": "Boiler Loop 1",
+                                                        "primary_airflow": 2359.737215999999,
+                                                        "supply_design_cooling_setpoint_temperature": 55,
```

Comment:
Do we also need the ```supply_design_cooling_setpoint_temperature```, ```supply_design_heating_setpoint_temperature``` and ```spaces``` keys  for thermal zone 2?

Reply:
yes, thanks for catching